tools/pfe-tools/tsconfig.json

```diff
@@ -7,14 +7,8 @@
     "dev-server/bundle.ts"
   ],
   "compilerOptions": {
-    "target": "es2022",
```

Comment:
We had to bump this to es2022 to support `Array.at`.  Do you think this will cause a regression at all?

Reply:
Might. Let's put it back